### PR TITLE
added basic examine functionality

### PR DESCRIPTION
**Pull request by takhlaq**
- todo: fix gear models not displaying

### CODE REVIEW

File: src/servers/sapphire_zone/Network/Handlers/ClientTriggerHandler.cpp [hunk 1]
```diff
@@ -118,6 +120,40 @@ void Core::Network::GameConnection::clientTriggerHandler( const Packets::FFXIVAR
         player.getCurrentAction()->setInterrupted();
       break;
     }
+    case ClientTriggerType::Examine:
```
Comment (SapphireMordred) [inline src/servers/sapphire_zone/Network/Handlers/ClientTriggerHandler.cpp]: Might wanna start putting big blocks like this into extra functions.

Comment (takhlaq) [reply src/servers/sapphire_zone/Network/Handlers/ClientTriggerHandler.cpp]: i was considering creating an examine packet class, yea

File: src/servers/sapphire_zone/Network/Handlers/ClientTriggerHandler.cpp [hunk 2]
```diff
@@ -40,6 +42,59 @@ using namespace Core::Network::Packets;
...
+      memcpy( packet->data().look, pPlayer->getLookArray(), sizeof( packet->data().look ) );
+      packet->data().models[ 0 ] = pPlayer->getModelForSlot( Common::GearSetSlot::Head );
+      packet->data().models[ 1 ] = pPlayer->getModelForSlot( Common::GearSetSlot::Body );
+      packet->data().models[ 2 ] = pPlayer->getModelForSlot( Common::GearSetSlot::Hands );
+      packet->data().models[ 3 ] = pPlayer->getModelForSlot( Common::GearSetSlot::Legs );
+      packet->data().models[ 4 ] = pPlayer->getModelForSlot( Common::GearSetSlot::Feet );
+
+      // todo: main/sub/other stuff too
+
+      for( auto i = 0; i < Common::GearSetSlot::SoulCrystal + 1; ++i)
```
Comment (SapphireMordred) [inline src/servers/sapphire_zone/Network/Handlers/ClientTriggerHandler.cpp]: How dare you miss a space

**Comment by takhlaq** [comment]
this pr also fixed playerspawn not showing accessories

**Pull request merged**